srcpkgs/samtools/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'samtools'
+pkgname=samtools
+version=1.13
+revision=1
+build_style=gnu-configure
+configure_args="--with-htslib=system"
+makedepends="htslib-devel zlib-devel ncurses-devel"
+checkdepends="perl"
+short_desc="Reading/writing/editing/indexing/viewing SAM/BAM/CRAM format"
```

Comment:
I'd say "read/write/edit/index/view SAM/BAM/CRAM files" is clearer? or "format files"

Reply:
I'd rather preserve the [original wording](https://www.htslib.org).

Reply:
I don't think using "reading" instead of "read" fits well for a package description... Would you agree to changing the verbs, at least?

Reply:
Okay